src/homeassistant.cpp

```diff
@@ -592,15 +592,34 @@ void HomeAssistant::sendCommand(const QString &type, const QString &entity_id, i
         RemoteInterface *remoteInterface = static_cast<RemoteInterface *>(entity->getSpecificInterface());
         QVariantList commands = remoteInterface->commands();
         QStringList remoteCodes = findRemoteCodes(entity->getCommandName(command), commands);
+        QString remoteDevice = findRemoteDevice(entity->getCommandName(command), commands);
 
         if (remoteCodes.length() > 0) {
             QVariantMap data;
+
+            if(remoteDevice.length() > 0) {
```

Comment:
Please fix cpplint issue:
`src/homeassistant.cpp:600:  Missing space before ( in if(  [whitespace/parens] [5]`